include/chibi/sexp.h

```diff
@@ -1680,6 +1680,8 @@ sexp sexp_finalize_dl (sexp ctx, sexp self, sexp_sint_t n, sexp dl);
 #define sexp_current_source_param
 #endif
 
+SEXP_API void sexp_add_static_libraries(struct sexp_library_entry_t* libraries);
```

Comment:
Can you add a comment that libraries must outlive all Chibi uses?

Reply:
I also added a few words to explain its use, as it will be tricky for prospective users to figure it out by inspecting the code.  Let me know if this is not desirable.